src/cow-react/modules/limitOrders/const/trade.ts

```diff
@@ -1,4 +1,6 @@
-import { Percent } from '@uniswap/sdk-core'
+import { Percent, Fraction } from '@uniswap/sdk-core'
 
 export const LIMIT_ORDER_SLIPPAGE = new Percent(0)
 export const LOW_RATE_THRESHOLD_PERCENT = -5
+export const MARKET_RATE_ADJUSTMENT = new Fraction(1, 1000) // 0.1%
+export const ONE = new Fraction(1)
```

Comment:
Can you refactor this part to use the same `ONE`?

https://github.com/cowprotocol/cowswap/blob/b366fa833887f6a86dacf93f425e2dc6f250c811/src/utils/calculateSlippageAmount.ts#L4